src/track/trackmetadatataglib.cpp

```diff
@@ -43,7 +49,7 @@ static_assert(sizeof(wchar_t) == sizeof(QChar), "wchar_t is not the same size th
 #include <taglib/mp4file.h>
 #include <taglib/flacfile.h>
 #include <taglib/vorbisfile.h>
-#if TAGLIB_HAS_OPUSFILE
+#if (TAGLIB_HAS_OPUSFILE)
```

Comment:
Works as expected: #if (!condition)
Did not work: #if !condition

This is the reason why I put parantheses around all conditions in #if 
preprocessor statements

On 13.07.2016 23:44, Daniel Schürmann wrote:
>
> In src/track/trackmetadatataglib.cpp 
> <https://github.com/mixxxdj/mixxx/pull/981#discussion_r70714717>:
>
> > @@ -43,7 +49,7 @@ static_assert(sizeof(wchar_t) == sizeof(QChar), "wchar_t is not the same size th
> >  #include <taglib/mp4file.h>
> >  #include <taglib/flacfile.h>
> >  #include <taglib/vorbisfile.h>
> > -#if TAGLIB_HAS_OPUSFILE
> > +#if (TAGLIB_HAS_OPUSFILE)
>
> I think it is in the responds of the #define statement to proper ( )
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/981/files/83633d64c0c9d7095376c786539248a194af434c#r70714717>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ABZ-aib1o2nrzLmhZzZk1_qiDycxhjidks5qVVw9gaJpZM4JLylV>.
>